internal/pkg/policy/monitor.go

```diff
@@ -125,6 +126,11 @@ LOOP:
 func (m *monitorT) process(ctx context.Context) error {
 	policies, err := m.policyF(ctx, m.bulker, dl.WithIndexName(m.policiesIndex))
 	if err != nil {
+		if errors.Is(err, es.ErrIndexNotFound) {
+			err = nil
```

Comment:
Debug log?

Reply:
Also return here immediately? There won't be any policies yet.

Reply:
don't know what I was thinking yesterday. will update.